doc/source/conf.py

```diff
@@ -306,11 +311,17 @@
         "json_url": f"https://{cname}/versions.json",
         "version_match": switcher_version,
     },
-    "use_meilisearch": {
-        "api_key": os.getenv("MEILISEARCH_PUBLIC_API_KEY", ""),
-        "index_uids": {
-            f"pymapdl-v{switcher_version.replace('.', '-')}": "PyMAPDL",
-        },
+    # "use_meilisearch": {
+    #     "api_key": os.getenv("MEILISEARCH_PUBLIC_API_KEY", ""),
+    #     "index_uids": {
+    #         f"pymapdl-v{switcher_version.replace('.', '-')}": "PyMAPDL",
+    #     },
+    # },
```

Comment:
Just delete this then?

Reply:
Good point, I will clean the `conf.py` file